packages/firebase_auth/firebase_auth/lib/src/password_policy/password_policy_status.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright 2025, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'password_policy.dart';
+
+class PasswordPolicyStatus {
+  bool status;
```

Comment:
is this supposed to be `isValid`? https://firebase.google.com/docs/reference/js/auth.passwordvalidationstatus?_gl=1*sgxowe*_up*MQ..*_ga*MzAzMTUyMTE1LjE3NTI3NDQxODk.*_ga_CW55HF8NVT*czE3NTI3NDQxODkkbzEkZzAkdDE3NTI3NDQxOTQkajU1JGwwJGgw#passwordvalidationstatusisvalid

I think we should match the properties on the JS SDK for the API exposed to the end user.